.github/workflows/build-test.yml

```diff
@@ -202,11 +223,44 @@ jobs:
           name: vercors-debian-package
       - run: "sudo dpkg -i *.deb"
       - run: ./util/veymontTesting/veymont-2stage-tests.sh
+  
+  integration-test-wiki:
+    needs: build-test
+    runs-on: ubuntu-latest
+    steps:
+      # Need to check out the repo for the "examples" directory
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "11"
+          java-package: jre
+      - name: Install wiki script dependencies
+        working-directory: util/wiki
+        run: |
+          sudo apt install pandoc
+          pip3 install -r requirements.txt
+      - name: Extract test cases from wiki
+        run: "python3 util/wiki/generate_wiki_pdf.py -c wiki_cases"
+      - name: Download VerCors binary
+        uses: actions/download-artifact@v2
+        with:
+          name: vercors-debian-package
+      - run: "sudo dpkg -i *.deb"
+      - run: vercors --silicon examples/manual/fibonacci.pvl
+      - run: vercors --test=wiki_cases --test-workers=2 --tool carbon --tool silicon --tool veymont --exclude-suite=slow,medium,problem-fail --progress --actions-test-output --enable-test-coverage --coverage-output-file=jacoco_wiki_tests.xml
+      - name: Archive Java Code Coverage (jacoco_wiki_tests.xml)
+        uses: actions/upload-artifact@v2
+        with:
+          name: vercors-jacoco-xml
+          retention-days: 1
+          path: |
+            jacoco*.xml
```

Comment:
Perhaps this should run without coverage?

Reply:
Perhaps it's also advisable to make a warning that changes to this sequence should be synchronized with the one in wiki-snippet-test

Reply:
Hmm, I guess in that case the example/test suite should be improved to also cover that part, after which the coverage from the wiki is no longer relevant, so indeed you are right! I'll remove it.